ember-cli-build.js

```diff
@@ -8,7 +8,13 @@ module.exports = function(defaults) {
       includePolyfill: true,
```

Comment:
```suggestion
      includePolyfill: false,
```

? While we're in here?